incubator/hnc/pkg/controllers/setup.go

```diff
@@ -11,6 +12,21 @@ import (
 	"github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/forest"
 )
 
+// Counters of total/current number of hier/object reconciliation attempts.
+var (
+	hcTot int32
```

Comment:
Are we not worried about overflowing here? Any reason to not use uint32 so we at least overflow to 0? I mean, it is highly unlikely we hit these limits during a perf test so we probably just don't care? How many reconciles to we anticipate per minute during a test? Will a long running soak test get near these limits?

Reply:
I doubt we're going to run anywhere near to 2B reconciles - even at 100 reconciles a second (which is way above what I think we can support), that's 240 days, which is a pretty long soak test!

I dunno - is uint32 better because it overflows to a possibly-valid value, or is int32 better because it overflows to an obviously-bad value? :)

Reply:
> I dunno - is uint32 better because it overflows to a possibly-valid value, or is int32 better because it overflows to an obviously-bad value? :)

Haha fair. I have a feeling this is like asking if you prefer emacs over vim...

> I doubt we're going to run anywhere near to 2B reconciles - even at 100 reconciles a second (which is way above what I think we can support), that's 240 days, which is a pretty long soak test!

Agreed, but I can't just let an unhandled overflow through without asking :)